app/code/Magento/Wishlist/Api/data/WishlistInterface.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Magento\Wishlist\Api\data;
+
+interface WishlistInterface
```

Comment:
1. Some setters for fields are missing 
2. Use strict types for methods 
3. Setter and getter for extension attributes are missing

Reply:
I've added setter for customerId, with 2, do you mean declare('strict_types') or that I should use typehints